test/integration/testdata/mysql.yaml

```diff
@@ -26,6 +26,13 @@ spec:
       containers:
       - image: mysql:5.6
         name: mysql
+        resources:
+          requests:
+            memory: "512Mi"
+            cpu: "500m"
+          limits:
+            memory: "1Gi"
+            cpu: "1000m"
```

Comment:
do u mind clarifying how you got these numbers? is there a chance we could go lower  ? would you please try running the functional test with "Lowest" possible CPU and memory limits before mysql breaks ?
to run funcitonal you can run:

"make functional"

Reply:
For the following configurations(minimum resources) the tests will get passed.
```
        resources:
          limits:
            memory: "512Mi"
            cpu: "1000m"
```

Reply:
why the request limits were deleted ?
1000m cpu seems a lot, have u tried lower and it failed?

Reply:
Yuh, I've tried with 500m. But it fails the test sir. I'll check some values between 500-1000m. May be we could be able to reduce the CPU limit

Reply:
yes please lets try with less than 1000m cpu and see what is min that works, and add +100 to that absolute min